### PR TITLE
Intro to Proportional control

### PR DESCRIPTION
**Pull request by AnselChang**

### CODE REVIEW

File: course/robot_control/intro_to_prop_control.rst [hunk 1]
```diff
@@ -1,47 +1,85 @@
 Introduction to Proportional Control
 ====================================
 
-Introduction:
--------------
+What is Proportional Control?
+-----------------------------
 
-While the on-off controller works, there are issues that can be solved using another type of controller.
+Imagine you're driving a car and you want to keep at a steady speed. If you're going too slow,
```
Comment (Kreeevin) [inline course/robot_control/intro_to_prop_control.rst:7]: Will need to replace this analogy down the line

**Review by Kreeevin — APPROVED** [review]

File: course/robot_control/prop_distance_tracker.rst [hunk 2]
```diff
@@ -22,7 +22,7 @@ is also "correct". The distinction in sign is simply whichever makes more sense
 we would want to drive forward 10cm, so we would want a positive error to make our motors spin forward.
 
 **Control Output**: In this case, this is our motor effort. This is because we want to drive with a speed proportional
-to the distance error. As a reminder for P control, this will be calculated as :code:`motor_effort = Kp * error`.
+to the distance error. As a reminder for P control, this will be calculated as :code:`motorEffort = Kp * error`.
```
Comment (Kreeevin) [inline course/robot_control/prop_distance_tracker.rst:25]: why the change to camel case here? I notice the blockly also uses it, but I thought we were sticking to python snake_case whenever possible?

File: course/robot_control/prop_distance_tracker.rst [hunk 3]
```diff
@@ -52,3 +52,36 @@ Let's start by defining our proportional gain and our set point:
...
+                drivetrain.set_effort(motorEffort, motorEffort)
+                time.sleep(0.05)
+
+    .. tab-item:: Blockly
+
+        .. image:: media/pcode.png
+            :width: 500
+
+Each iteration of the loop consists of the following steps:
+    #. Poll the rangefinder to get the current distance
```
Comment (Kreeevin) [inline course/robot_control/prop_distance_tracker.rst:76]: Use of the word Poll is a small nitpick, but may not make sense to non-stem/robotics people

File: course/robot_control/prop_distance_tracker.rst (hunk 2 above)
Comment (Kreeevin) [reply course/robot_control/prop_distance_tracker.rst:25]: this actually applied to everywhere in the section. What is being done elsewhere in the curriculum?

**Review by Kreeevin — CHANGES_REQUESTED** [review]
You're really testing the boundaries of 'acceptable' with this, but I admire your boldness.

File: course/robot_control/prop_distance_tracker.rst (hunk 3 above)
Comment (AnselChang) [reply course/robot_control/prop_distance_tracker.rst]: fixed

File: course/robot_control/prop_distance_tracker.rst (hunk 2 above)
Comment (AnselChang) [reply course/robot_control/prop_distance_tracker.rst]: fixed

**Review by Kreeevin — APPROVED** [review]
Wow! Your work is absolutely amazing! It's like a synchronization rate off the charts, just pure excellence! You've really piloted this project to greatness and left the Angels in awe. Keep it up, and you'll be the Eva pilot of success!

**Pull request merged**